routes/home.js

```diff
@@ -0,0 +1,30 @@
+const blogPosts = require("../blogPosts.js");
+
+function get(request, response) {
+    let posts = '';
+    for (const post of Object.values(blogPosts)) {
+        posts += `
+        <div class="blog-post"><h3>${post.user}</h3> <p>${post.message}</p></div>
```

Comment:
Is this where we would implement the hack-proofing? 👀 Though not sure how much damage people can do with a username and message...

Reply:
I think the xss protection comes in the form submission post.